src/javascripts/helpers/data/authData.js

```diff
@@ -13,11 +14,14 @@ const checkLoginStatus = () => {
       authDiv.addClass('hide');
       homeDiv.removeClass('hide');
       logoutButton.removeClass('hide');
+
+      $('#crew').show(crewList.buildCrew);
```

Comment:
so you'll want to do the same thing that's happening on line 19: `crewList.buildCrew()` instead of this show thing

Reply:
@luketlancaster it should be there now